src/base.py

```diff
@@ -29,4 +37,13 @@ def change_to_base(n: int, b: int) -> str:
     '1F'
     """
     assert 2 <= b <= 16
-    return ''  # FIXME: return n in the right base
+    lst = []
+    while n >= b: 
```

Comment:
Have you tested this for negative `n`? E.g., with `n == -12` and `b == 3`. The right answer should be `-110` (since $-(1\cdot 3^2 + 1\cdot 3^1 + 0\cdot 3^0) = -12$ ), but $n < b$ (-12 < 3) so we never enter the `while` loop.